benchmarks/auth/Makefile

```diff
@@ -50,6 +50,12 @@ auth-python-lambda-image: docker/Dockerfile.Lambda python/server.py
 	-f docker/Dockerfile.Lambda \
 	$(ROOT) --load
 
+auth-python-lambda-image: docker/Dockerfile.Lambda python/server.py
+	DOCKER_BUILDKIT=1 docker buildx build \
+	--tag $(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com/auth-python-lambda:latest \
+	--target authPythonLambda \
+	-f docker/Dockerfile.Lambda \
+	$(ROOT) --load
```

Comment:
Just out of curiosity `--load` do you know what is it for?

Reply:
The way I understand it, `--load` is shorthand for `--output=type=docker`.
As per the [docs](https://docs.docker.com/engine/reference/commandline/buildx_build/#output), `docker build` exports built container images to `docker images` while in case of `docker buildx build` you can configure this step with the `output` option. To ensure the default behaviour (that it shows up in `docker images`) we use `--output=type=docker`.